chapters/data-analysis.tex

```diff
@@ -1,292 +1,487 @@
+The underlying data structure does not change.
+The cleaned data set should contain only the variables collected in the field.
+No modifications to data points are made at this stage, except for corrections of mistaken entries.
+
+Cleaning is probably the most time consuming of the stages discussed in this chapter.
+This is the time when you obtain an extensive understanding of  the contents and structure of the data that was collected.
+Explore your data set using tabulations, summaries, and descriptive plots.
+You should use this time to understand the types of responses collected, both within each survey question and across respondents.
+Knowing your data set well will make it possible to do analysis.
+
+
+% Deidentification ------------------------------------------------------------------
+The initial input for data cleaning is the raw data.
+It should contain only materials that are received directly from the field.
+They will invariably come in a host of file formats and nearly always contain personally-identifying information.\index{personally-identifying information}
+These files should be retained in the raw data folder \textit{exactly as they were received}.
+The folder must be encrypted if it is shared in an insecure fashion,\sidenote{\url{https://dimewiki.worldbank.org/wiki/Encryption}}
 and it must be backed up in a secure offsite location.
 Everything else can be replaced, but raw data cannot.
 Therefore, raw data should never be interacted with directly.
 
-Instead, the first step upon receipt of data is \textbf{de-identification}.\sidenote{\url{https://dimewiki.worldbank.org/wiki/De-identification}}
-\index{de-identification}
-There will be a code folder and a corresponding data folder
-so that it is clear how de-identification is done and where it lives.
-Typically, this process only involves stripping fields from raw data,
-naming, formatting, and optimizing the file for storage,
-and placing it as a \texttt{.dta} or other data-format file
-into the de-identified data folder.
-The underlying data structure is unchanged:
-it should contain only fields that were collected in the field,
-without any modifications to the responses collected there.
-This creates a survey-based version of the file that is able
-to be shared among the team without fear of data corruption or exposure.
-Only a core set of team members will have access to the underlying
-raw data necessary to re-generate these datasets,
-since the encrypted raw data will be password-protected.
-The de-identified data will therefore be the underlying source
-for all cleaned and constructed data.
-It will also become the template dataset for final public release.
-
-%------------------------------------------------
+Secure storage of the raw\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data\_Security}}
+data means access to it will be restricted even inside the research team.
+Loading encrypted data multiple times it can be annoying.
```

Comment:
_Loading encrypted data multiple times it can be annoying._ A few things:
* there is a typo in _ it can be_
* multiple is not the right work here. should we say _frequently_ or _regularly_?
* Annoying is a negatively charged word. What about: _can be disruptive to the work flow_?

Reply:
In c80a095:
> Loading encrypted data frequently can be disruptive to the workflow.